src/main/java/ee/tuleva/onboarding/mandate/Mandate.java

```diff
@@ -59,4 +59,19 @@ protected void onCreate() {
         return Optional.ofNullable(futureContributionFundIsin);
     }
 
+    public Map<String, List<FundTransferExchange>> getPrintableFundExchangeStructure() {
```

Comment:
Maybe rename to `getGroupedExchanges` or something along those lines since it has nothing to do with _printing_.